### PR TITLE
Artifact rename

### PR DESCRIPTION
**Pull request by seglo**

### CODE REVIEW

File: project/Common.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ object Common extends AutoPlugin {
   override def requires = JvmPlugin && HeaderPlugin
 
   override def globalSettings = Seq(
-    organization := "com.lightbend.akka",
+    organization := "org.pekko",
```
Comment (nvollmar) [inline project/Common.scala:24]: `org.apache.pekko`?

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by pjfanning — APPROVED** [review]
lgtm

**Review by pjfanning — CHANGES_REQUESTED** [review]
actually, the issue that @nvollmar highlighted in project/Common.scala should probably be looked at

**Pull request merged**